tests/vector/test_spaces.py

```diff
@@ -116,9 +116,11 @@ def test_batch_space_custom_space(space, expected_batch_space_4):
 def test_iterate(space, batch_space):
     items = batch_space.sample()
     iterator = iterate(batch_space, items)
+    final_i = 0
```

Comment:
Just do `i=0` before the loop and get rid of `final_i`

Reply:
Done